portfolio-site-dump.sql

```diff
@@ -0,0 +1,319 @@
+-- MySQL dump 10.16  Distrib 10.1.44-MariaDB, for debian-linux-gnu (x86_64)

```

Comment:
wonder if this can be formatted better so database changes would be easier to understand